summarize/src/main.rs

```diff
@@ -52,20 +65,26 @@ fn main() -> Result<(), Box<std::error::Error>> {
     let total_time = results.total_time.as_nanos() as f64;
 
     for query_data in results.query_data {
-        table.add_row(row![
-            query_data.label,
-            format!("{:.2?}", query_data.self_time),
-            format!("{:.3}", ((query_data.self_time.as_nanos() as f64) / total_time) * 100.0),
-            format!("{}", query_data.invocation_count),
-            format!("{}", query_data.number_of_cache_hits),
-            format!("{:.2?}", query_data.blocked_time),
-            format!("{:.2?}", query_data.incremental_load_time),
-        ]);
+
+        let percent = (query_data.self_time.as_nanos() as f64) / total_time * 100.0;
+
+        if percent > percent_above {
```

Comment:
The data is ordered by self-time descending, so once this condition is hit we can actually `break` out of the loop entirely since all remaining results will be less than the current value.